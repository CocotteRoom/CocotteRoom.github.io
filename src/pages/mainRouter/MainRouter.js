import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import { Home } from './../home/Home';
import { Recipe } from './../recipe/Recipe';

export const MainRouter = () => (
  <BrowserRouter>
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/recipe" component={Recipe} />
    </Switch>
  </BrowserRouter>
);
