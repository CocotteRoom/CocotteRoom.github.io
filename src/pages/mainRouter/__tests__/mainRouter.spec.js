import React from 'react';
import { shallow } from 'enzyme';
import { MainRouter } from '../MainRouter';

describe('MainRouter', () => {
  const getWrapper = () => shallow(<MainRouter />);

  it('should match snapshot', () => {
    expect(getWrapper()).toMatchSnapshot();
  });
});
