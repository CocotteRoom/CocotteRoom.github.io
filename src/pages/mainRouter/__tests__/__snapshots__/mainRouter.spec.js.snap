// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainRouter should match snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/recipe"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
